src/utils/replaceWithExactPath.js

```diff
@@ -0,0 +1,14 @@
+const which = require('which');
+
+function replace(command) {
+    return command
+        .split('&&')
```

Comment:
commands can also be split by `;`

Reply:
Also there can be new line i.e. `\n` 
Please take these also into account

Reply:
Got it, fixed